listener/iot-connection.ts

```diff
@@ -28,12 +29,22 @@ export async function getConnection(
     logError('No IoT endpoint could be found');
     throw new Error('IoT Endpoint address not found');
   }
+  const region = process.env['AWS_REGION'] || process.env['AWS_PROFILE_REGION'];
```

Comment:
Why is this check necessary? You do not use `region` variable.

Reply:
Yeah it should of course be used in the device call, was perhaps a bit tired when I added that yesterday 😅 I'll fix it when I get to work.

Reply:
But that would require that you must always set `AWS_REGION` or `AWS_PROFILE_REGION`. 
People usually do not set that. Or am I missing something?

Reply:
the iot connection requires a region to be set, otherwise it will default to us-east-1. Let me add another fallback to CDK_DEFAULT_REGION as well.

Reply:
The iot connection does require region to be set, otherwise it will pick us-east-1. We could perhaps put it as a config variable and set it in the output, but that would require people to update their test code.

Reply:
Hmm, there rings a bell that the region is part of the url, let me check.

Reply:
Yeah I can get it from te iot endpoint instead. This won't support any kind of custom iot endpoint setup but we're probably okay with that. We can always add something more complex to configure that kind of scenario at a later point. But I think the underlying iot endpoint should always be present, even if someone configured something in Route53 so it should still work.

Reply:
Great!
Thank you for doing this.

Reply:
@ServerlessLife no problem at all, and thanks for taking the time to look through the PR properly! :)